integration/local-gadget/helpers_test.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"os/exec"
+	"time"
+
+	"google.golang.org/grpc"
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+var ErrNoCommandProvided = errors.New("no command provided")
+
+func run(commands ...string) ([]byte, error) {
```

Comment:
It would be nice to use [command](https://github.com/kinvolk/inspektor-gadget/blob/main/integration/command.go#L36) here. But I wanted to avoid adding multi module dependency like [this](https://github.com/kinvolk/inspektor-gadget/pull/945#discussion_r972908078). Also #945 will be super helpful for JSON validation.